biz.aQute.repository/src/aQute/bnd/repository/p2/provider/P2Indexer.java

```diff
@@ -97,7 +98,7 @@ File get(String bsn, Version version, Map<String,String> properties, DownloadLis
 		if (link.isFile())
 			Files.delete(link.toPath());
 
-		Files.createSymbolicLink(link.toPath(), source.toPath());
+		IO.createSymbolicLinkOrCopy(link, source);
```

Comment:
Hmmm, this can get busy on a windows system since we always delete the file above and then recopy it. :-(

Deleting the symlink and recreating is (hopefully) cheap. But always deleting and recopying a potentially large file for no reason is wasteful.

@gamerson's original attempt at this tried to detect if the target file was already the proper file to shortcut the delete/copy work. So we may need something that computes the sha of the source and target and deletes/copies only when the shas differ. I guess the question is then whether the the sha computation is more costly than always delete/copy?